mantidimaging/gui/windows/live_viewer/presenter.py

```diff
@@ -114,12 +114,13 @@ def load_image(image_path: Path) -> np.ndarray:
         Load a .Tif, .Tiff or .Fits file only if it exists
         and returns as an ndarray
         """
+        image_data = np.array([])
```

Comment:
I think would be to have an `else` that raises an exception if it is an unrecognised file time.

Reply:
Looking at where this method is referenced, I will raise as a value error so it is picked up nicely in `display_image()`